packages/integrations/cloudflare/README.md

```diff
@@ -191,6 +191,42 @@ export default defineConfig({
 });
 ```
 
+## WASM module imports
+
+`wasmModuleImports: boolean`
+
+Cloudflare has native support for importing `.wasm` files [directly as ES modules](https://github.com/WebAssembly/esm-integration/tree/main/proposals/esm-integration). To enable importing them as modules in the cloudflare build and astro dev server, add `wasmModuleImports: true` to your config.
+
+```diff
+import {defineConfig} from "astro/config";
+import cloudflare from '@astrojs/cloudflare';
+
+export default defineConfig({
+    adapter: cloudflare({
++       wasmModuleImports: true
+    }),
+	  output: 'server'
+})
+```
+
+Once enabled, you can import a web assembly module in astro with a `.wasm?module` import. The integration supports WASM module imports in both server and hybrid mode for a consistent development experience.
+
+```javascript
+// pages/add/[a]/[b].js
+import mod from '../util/add.wasm?module';
+
+// instantiate ahead of time to share module
+const addModule: any = new WebAssembly.Instance(mod);
+
+export async function GET(context) {
+  const a = Number.parseInt(context.params.a!);
+	const b = Number.parseInt(context.params.b!);
+  return new Response(`${a} + ${b} = ${addModule.exports.add(a, b)}`);
+}
+```
+
+
```

Comment:
```suggestion
The Cloudflare integration supports WASM module imports in both `server` and `hybrid` mode for a consistent development experience.
```
This seems more appropriate as a "final note" rather than interrupting explaining usage.

Reply:
Reading this sentence over, it felt redundant. I would imagine that user expectation is that the feature works out of the box. 

Updated instead to give an idea of a more exciting potential use case for Wasm. I like the addition example because its simple and intuitive, but its not a practical use Wasm.